### PR TITLE
Fixed mention of Kafka Connect

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

* Updated Kafka references to mention full name of connector.
* Edited entire page for word usage, grammar, style, etc.
* Checked, updated, and removed unused links

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/577

### CODE REVIEW

File: timescaledb/how-to-guides/ingest-data.md [hunk 1]
```diff
@@ -1,100 +1,79 @@
...
-be accessed by connecting to Timescale directly. It also offers other
-native time-series capabilities, such as automatically
+Prometheus is used to monitor infrastructure metrics. It scrapes any endpoints
+that can expose metrics in a compatible format. The metrics are stored in
+Prometheus, and you can query them using the PromQL query language. Prometheus
+is not intended to be used for long-term metrics storage, but it supports a
+variety of remote storage solutions that can be used for that purpose.
+
+TimescaleDB can use Prometheus as a remote store for long-term metrics, by using
+[Promscale][promscale]. Promscale supports both PromQL and SQL queries. PromQL
```
Comment (charislam) [inline timescaledb/how-to-guides/ingest-data.md:22]: Don't know if this is just a problem with the dev environment, but this gave me a 403 error. All the other links work.

File: timescaledb/how-to-guides/ingest-data.md [hunk 2]
```diff
@@ -1,100 +1,79 @@
...
-TimescaleDB, you can download the
-[StreamSets Data Collector][streamsets-data-collector] and get started with
-this [tutorial][tutorial-streamsets].
+aggregate views, downsampling, data gap-filling, and interpolation.
+Additionally, Promscale supports Grafana using [Prometheus][prometheus-grafana]
+and [PostgreSQL][postgres-grafana] data sources.
+
+## Telegraf
+Telegraf collects, processes, aggregates, and writes metrics. Telegraf is highly
+extensible, and has over two hundred plugins for gathering and writing different
```
Comment (charislam) [inline timescaledb/how-to-guides/ingest-data.md:33]: What's our style guide? I'd lean towards numerals for "200" (marginally better readability, follows [Google's style guide](https://developers.google.com/style/numbers), which is what I think we lint against?) but not a strongly held opinion

**Review by charislam — APPROVED** [review]

File: timescaledb/how-to-guides/ingest-data.md (hunk 2 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/ingest-data.md:33]: Let's roll with Google. I think I was harking back to Chicago on that one 😉

File: timescaledb/how-to-guides/ingest-data.md [hunk 3]
```diff
@@ -1,100 +1,79 @@
...
```
Comment (Loquacity) [inline timescaledb/how-to-guides/ingest-data.md:33]:
```suggestion
extensible, and has over 200 plugins for gathering and writing different
```

File: timescaledb/how-to-guides/ingest-data.md (hunk 1 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/ingest-data.md:22]: @jacobprall ??

Comment (jacobprall) [reply timescaledb/how-to-guides/ingest-data.md:22]: @charislam change to /promscale/:currentVersion:/ for the URL

**Pull request merged**